.renovaterc.json5

```diff
@@ -127,5 +127,11 @@
         "\"openApiSupportedVersion\": \"~(?<currentValue>[^\"]+)\""
       ]
     }
-  ]
+  ],
+  "github-actions": {
+    "fileMatch": [
+      "(^|/)(__dot__github|workflow-templates|\\.(?:github|gitea|forgejo)/(?:workflows|actions))/.+\\.ya?ml$",
```

Comment:
`__dot__` ? Would not be `[.]` or `\\.` instead ?

Reply:
here I am trying to override the default `fileMatch` [value](https://docs.renovatebot.com/modules/manager/github-actions/) so that it goes over my templates that are located in a `__dot__github` folder